prog/probe.go

```diff
@@ -158,6 +164,7 @@ func probeMain(flags probeFlags, targets []appclient.Target) {
 		SpyProcs:     flags.spyProcs,
 		UseConntrack: flags.useConntrack,
 		WalkProc:     flags.procEnabled,
+		EbpfEnabled:  flags.ebpfEnabled,
```

Comment:
we could track several things with ebpf: processes, connection, ios...

but this PR is only about tracking connections. We should have a better name for the option than a generic "ebpf", we should mention "connections" somewhere.

Ditto for the "EbpfTracker"